requirements/test-requirements.txt

```diff
@@ -19,3 +19,4 @@ plotly
 kaleido
 # The latest flask version requires werkzeug>=2.2.0
 werkzeug>=2.2.0
+virtualenv
```

Comment:
`virtualenv` is included in the extra dependencies.

https://github.com/mlflow/mlflow/blob/2305eb5c2397f5739a10bda559790be0b5a05ad9/setup.py#L147